src/deployconfig.jl

```diff
@@ -274,15 +283,19 @@ function deploy_folder(cfg::GitHubActions; repo, devbranch, push_preview, devurl
     actor_ok = haskey(ENV, "GITHUB_ACTOR")
     all_ok &= actor_ok
     println(io, "- $(marker(actor_ok)) ENV[\"GITHUB_ACTOR\"] exists")
-    ## GITHUB_TOKEN should exist (just check here and extract the value later)
-    token_ok = haskey(ENV, "GITHUB_TOKEN")
-    all_ok &= token_ok
-    println(io, "- $(marker(token_ok)) ENV[\"GITHUB_TOKEN\"] exists")
+    # ## GITHUB_TOKEN should exist (just check here and extract the value later)
+    # token_ok = haskey(ENV, "GITHUB_TOKEN")
+    # all_ok &= token_ok
+    # println(io, "- $(marker(token_ok)) ENV[\"GITHUB_TOKEN\"] exists")
+    ## DOCUMENTER_KEY should exist (just check here and extract the value later)
+    key_ok = haskey(ENV, "DOCUMENTER_KEY") || haskey(ENV, "GITHUB_TOKEN")
```

Comment:
Could we print a warning here if the user only sets `GITHUB_TOKEN` that says something like "GitHub Pages may not update correctly, see #1177"?

Reply:
Done